less/inc/widgets.less

```diff
@@ -210,7 +210,7 @@
   }
 }
 .single.normal #sidebar .widget {
-  padding: 0 @baseline 0 0;
+  padding: 0 (@baseline / 2);
```

Comment:
this will also affect the two column layout (i think) so we may not want to do this...maybe